Extension.pm

```diff
@@ -203,6 +204,23 @@ sub buglist_column_joins {
     };
 }
 
+##########################################
+# Additional operations when creating bugs
+##########################################
+
+sub object_end_of_create{
+    my ($self, $args) = @_;
+    my $class  = $args->{'class'};
+    my $object = $args->{'object'};
+
+    my $cgi = Bugzilla->cgi;
+    my $pool_id = $cgi->param('pool_id');
+    if ($object->isa("Bugzilla::Bug")) {
+       return if (!defined $pool_id);
+       $object->set_pool_id($pool_id);
```

Comment:
Investigated this a bit, and it works because `post_bug.cgi` calls `bug->update()` after it has created the bug to set flags etc. So this kind of works as an unexpected side effect. A better way would probably be to get the Pool object and use `pool->add_bug($object)` like in the `object_end_of_update` hook.

Reply:
OK. I will fix it